### PR TITLE
Added examples for map_range and shift_in/shift_out

### PR DESCRIPTION
**Pull request by brentru**
added usage examples for map_range and shifting methods

### CODE REVIEW

File: examples/map_range_demo.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+'map_range_demo.py'.
+
+=================================================
+maps a number from one range to another
+"""
+import time
+import simpleio
+
+while True:
+    SENSOR_VALUE = 150
```
Comment (tannewt) [inline examples/map_range_demo.py]: The variables should be lower case. Travis runs pylint on examples with invalid-name off for this reason: https://github.com/adafruit/Adafruit_CircuitPython_SimpleIO/blob/master/.travis.yml#L29

Comment (brentru) [reply examples/map_range_demo.py]: Ok, variables switched to lower case.

**Review by tannewt — APPROVED** [review]
Looks good! Thank you!

**Comment by tannewt** [comment]
Sorry I didn't see this earlier.

**Pull request merged**